integration/playwright.config.ts

```diff
@@ -26,7 +26,7 @@ const config: PlaywrightTestConfig = {
       use: devices["Desktop Safari"],
     },
     {
-      name: "edge",
+      name: "msedge",
```

Comment:
I changed the name of this profile to match the name of the browser (like the others) so we can use this name to only install the required browser during CI with:

```
npx playwright install --with-deps ${{ matrix.browser }}
```

That fails for `edge` because playwright refers to it as `msedge`